docs/components/filmstrip/README.md

```diff
@@ -0,0 +1,86 @@
+---
+{
+  "title": "Filmstrip",
+  "title_metadata": false,
+  "layout_type": "LayoutArticle",
+  "summary": "",
+  "breadcrumbs": [
+    {
+      "text": "Components/"
+    }
+  ],
+}
+---
+
+
+<cdr-doc-table-of-contents-shell >
+
+# Overview
+    
+<cdr-banner type="info">
+  <template #icon-left>
+    <icon-information-fill />
+  </template>
+Filmstrip is a Cedar cookbook component.
+  <template #message-body>
+   Developers can choose from an optional range of subcomponents to create a custom version of it.
+  </template>
+</cdr-banner>
```

Comment:
This won't work as-is (pulling in other cedar components). (it's why the build is failing). I can assist